clusterman/migration/worker.py

```diff
@@ -283,8 +294,14 @@ def event_migration_worker(migration_event: MigrationEvent, worker_setup: Worker
     finally:
         if pool_lock_acquired:
             pool_lock.release()
-        # we do not reset the pool target capacity in case of pre-scaling as we
-        # trust the autoscaler to readjust that in a short time eventually
+        # we do not reset the pool target capacity in case of direct capacity changes
+        # as we trust the autoscaler to readjust that in a short time eventually
         if worker_setup.disable_autoscaling:
             logger.info(f"Re-enabling autoscaling for {migration_event.cluster}:{migration_event.pool}")
             enable_autoscaling(migration_event.cluster, migration_event.pool, SUPPORTED_POOL_SCHEDULER)
+        elif worker_setup.prescaling:
+            remove_capacity_offset(
+                migration_event.cluster,
+                migration_event.pool,
```

Comment:
We are using cluster, pool and scheduler as a key in some places. Should we check (and reject) if requested pool already has on-going migration?

Reply:
It's impossible for a pool to have concurrent migrations running, there's a lock preventing that.

Reply:
Rather, "impossible as long as you don't run more than one instance of the batch per cluster", which must not be done as many more things would break.

Reply:
As I understood we are checking it while worker running. I meant directly decline 2nd migration creation and respond to client via CLI. 
We are deleted (actually changing label to `STOP`) pending and in-progress migration events when we run `migrate-stop` command. This is a bit implicit.
But having multiple migration event and processing them one by one is cool.